apps/CMakeLists.txt

```diff
@@ -36,5 +36,7 @@ set(CMAKE_INSTALL_PREFIX /opt/rocm/mivisionx)
 if(OpenCL_FOUND AND miopengemm_FOUND AND miopen_FOUND)
     if(OpenCV_FOUND)
         add_subdirectory (cloud_inference/server_app)
+    else()
+        message("-- Cloud Inference Deps Not Found -- Cloud Inference application excluded")
```

Comment:
I think cloud inference dependencies are same as MIVisionX. Change to MIVisionX dependencies not found.